src/controller/java/BUILD.gn

```
@@ -229,6 +229,10 @@ shared_library("jni") {
     ldflags = [ "-Wl,--gc-sections" ]
   }
 
+  # Compatibility with 16K pages for Andropid 15+ support on 64-bit devices.
+  # See https://developer.android.com/guide/practices/page-sizes
+  ldflags += [ "-z,max-page-size=16384" ]
```

Should this be:

```suggestion
  ldflags += [ "-Wl,-z,max-page-size=16384" ]
```

For linker argument passing?

---

Android docs say `LOCAL_LDFLAGS += "-Wl,-z,max-page-size=16384"` so will apply this change.